common/src/api/external/error.rs

```diff
@@ -327,8 +327,8 @@ macro_rules! bail_unless {
     };
     ($cond:expr, $($arg:tt)+) => {
         if !$cond {
-            return Err($crate::api::external::Error::internal_error(&format!(
-                $($arg)*)))
+            Err($crate::api::external::Error::internal_error(&format!(
```

Comment:
This kind of strange change makes it possible to use `bail_unless!` in a situation where your function isn't returning `Error` per se, but something that impls `From<Error>`.